Packs/DeCYFIR/Integrations/decyfir/decyfir.py

```diff
@@ -0,0 +1,490 @@
+                                                                         api_param_query)
+
+        return json.dumps(return_data)
+
+    def prepare_incident_json(self, source_brand: str, alert_type: str, alert_subtype: str, name: str, date_val: str,
+                              severity: int, details: str, record_id: str) -> Dict[str, Any]:
+
+        # incident_owner = "Administrator"
+        occurred_date = dateparser.parse(date_val)
+        occurred = occurred_date.strftime(DATE_FORMAT) if isinstance(occurred_date, datetime) else None
+
+        return_data = {
+            "type": "" + f"{alert_type}",
+            "name": name,
+            "occurred": occurred,
+            # "owner": incident_owner,
+            "severity": severity,
+            "details": details,
+            "rawJSON": details,
+            "decyfirsubcategory": alert_subtype,
+            "decyfircategory": alert_type,
+            "category": alert_type,
+            "dbotMirrorId": record_id,
+            "sourceBrand": LABEL_DECYFIR,
+            "labels": [
+                {
+                    "type": "decyfircategory",
+                    "value": alert_type
+                },
+                {
+                    "type": "decyfirsubcategory",
+                    "value": alert_subtype
+                },
+                {
+                    "type": "incident_source_from",
+                    "value": source_brand
+                }
+            ]
+        }
+
+        return return_data
+
+    def prepare_incidents_for_attack_surface(self, json_data, alert_type: str, alert_subtype: str) -> List[dict]:
+        try:
+            incidents_json = []
+            for json_ in json.loads(json_data):
+                severity = self.get_severity(json_.get("risk_score"))
+                ip = json_.get("ip")
+                details = str(json.dumps(json_))
+                date_val = json_.get("alert_created_date")
+                uid = json_.get("uid")
+
+                domain: str = ""
+                if json_.get("sub_domain"):
+                    domain = json_.get("sub_domain")
+                domain = domain + ", " + json_.get("top_domain") if domain else json_.get("top_domain")
+
+                name = "DOMAIN : {}".format(domain) if domain else ""
+
+                if ip:
+                    name = name + "\n IP: {}".format(ip) if name else "IP: {}".format(ip)
+
+                if not name:
+                    name = "Asset: {}".format(json_.get("asset_name")) if json_.get("asset_name") else ""
+
+                incident_json = self.prepare_incident_json(LABEL_DECYFIR, alert_type, alert_subtype,
+                                                           name, date_val, severity, details, uid)
+                incidents_json.append(incident_json)
+
+            return incidents_json
+        except Exception as e:
+            return_error(str(e))
+            return []
```

Comment:
The exception will be caught in the main function.

```suggestion
            raise DemistoException(str(e))
```